F2000

Braiam23 opened the pull request:
Can you try to replace the .java model, see if it works please?
and in the case that the mod breaks, revert the change

Paneedah commented:
@SticksDeveloper please revierw this change if possible.

Koud-Wind commented:
I tried using these yesterday, and although I fixed some coding errors, there were errors in the F2000's sight and the aiming action still remained unchanged

Braiam23 commented:
> I tried using these yesterday, and although I fixed some coding errors, there were errors in the F2000's sight and the aiming action still remained unchanged

The only thing that I modified from the model is a little bit of code.
![image](https://github.com/Cubed-Development/Modern-Warfare-Cubed/assets/133471454/c02cd136-c11c-4297-a8ba-72e3868fe4e3)
